spec/features/user_delete_answer_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe 'UserDeleteAnswers', type: :feature do
+  describe 'User delete answer' do
+    context 'when registered user', js: true do
+      let(:user) { create(:user) }
+      let!(:answer) { create(:answer, user: user, question: create(:question)) }
+
+      before do
+        sign_in user
+        visit question_path(answer.question)
+      end
+
+      it 'has link Delete answer' do
+        within '.answers' do
+          expect(page).to have_link 'Delete'
+        end
+      end
+
+      it 'has delete current answer' do
+        within '.answers' do
+          delete_answer
+          expect(page).not_to have_content answer.body
+        end
+      end
+
```

Comment:
You can check if the answer has been deleted.

Reply:
That is the check. I think, the check if the answer has been deleted, held in controller test.

Reply:
This check checks the only page content.